apps/web/src/pages/Applications/ApplicationReviewPage/ApplicationReviewPage.tsx

```diff
@@ -393,41 +434,50 @@ const ApplicationReview = ({
         <Heading level="h3" data-h2-margin="base(0, 0, x1, 0)">
           {intl.formatMessage({
             defaultMessage: "Sign and submit",
-            id: "+sWqVY",
+            id: "fhgZRX",
+            description:
+              "Heading for sign and submit section of application review page.",
           })}
         </Heading>
         <div>
           <FormProvider {...methods}>
             <form onSubmit={methods.handleSubmit(handleSubmit)}>
               <p data-h2-margin-bottom="base(x1)">
                 {intl.formatMessage({
-                  defaultMessage: `"You made it! By signing your name, you confirm that:"`,
-                  id: "d0KeTK",
+                  defaultMessage: `You made it! By signing your name, you confirm that:`,
+                  id: "e05s3z",
+                  description:
+                    "Heading for sign and submit section of application review page.",
```

Comment:
This isn't a heading, perhaps change this description to (don't forget the fr.json!):

Instructions for sign and submit section of application review page.

Reply:
Thanks for all the suggestions ❤️! Fixed by [a972a4e](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6600/commits/a972a4ed7e18c3b9fb1392ec7b859fbc5815cc4c)